docs/reST/tut/MoveIt.rst

```diff
@@ -376,7 +378,7 @@ user asks us to stop. ::
   >>> while True:
   ...     for event in pygame.event.get():
   ...         if event.type in (QUIT, KEYDOWN):
-  ...             sys.exit()
+  ...             return
```

Comment:
Uhm I did not mean that this change should be a `return`. Here it should probably be `sys.exit()` itself because this is running in interactive prompt. 
What I meant was to make `if event.type in (QUIT, KEYDOWN):` to `if event.type == KEYDOWN:`